src/app/shared/context-resolver.service.ts

```diff
@@ -17,7 +17,11 @@ export class ContextResolver implements Resolve<Context> {
 
   constructor(private contextService: ContextService, private router: Router) {
     // The default place to navigate to if the context cannot be resolved is /home
-    this._lastRoute = '/_home';
+    this._lastRoute = '/_home'; // '/_error';
```

Comment:
for now we should keep things the way they are, when UX updated the flow we can update how this works